doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -420,6 +423,151 @@ an additional feature and still be conformant with Anuket.
 [here](../../../ref_model/chapters/chapter05.md#5.1) and hardware profile
 features [here](../../../ref_model/chapters/chapter05.md#5.4).
 
+
+<a name="3.2.2.1"></a>
+ ### 3.2.2.1 Kubernetes Networking Semantics
+The support for traditional network orchestration is non existent in Kubernetes as it is foremost a Platform as a Service (PaaS) environment and not an Infrastructure as a Service (Iaas) component. There is no network orchestration API, like Neutron in OpenStack, and there is no way to create L2 networks, instantiate network services such as L3aaS and LBaaS and then connect them all together as can be done using Neutron.
```

Comment:
Why wouldn't a PaaS have the ability to orchestrate what is provided by the underlying IaaS?

Reply:
> Why wouldn't a PaaS have the ability to orchestrate what is provided by the underlying IaaS?

It would.
I propose we don't mention PaaS here (as per suggestion I just made on line 429).